db/app/app-tests/app.test.js

```diff
@@ -114,3 +114,43 @@ describe('/api/getArticles', () => {
     })
   });
 });
+
+describe("/api/articles/:article_id/comments", () => {
+  test("200: should return a sorted array of all comments for specified article via the article_id, with the most recent first", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const {comments} = body
+        expect(comments).toBeSortedBy("created_at", {descending: true})
+        comments.forEach((comment) => {
+          expect(comment).toMatchObject(
+            {
+              comment_id: expect.any(Number),
+              votes: expect.any(Number),
+              created_at: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
+              article_id: expect.any(Number),
+            },
```

Comment:
Nice use of jest sorted here! 
Great use of the flexible jest sorter toMatchObject().

**As you are asserting inside a forEach() you should also make sure that the array is not empty.**

Reply:
Still need to do this Joe ☝️ 
Can you add an assertion to make sure this array of comments isn't empty